pkg/operator/ceph/config/conditions.go

```diff
@@ -117,28 +117,32 @@ func translatePhasetoState(phase cephv1.ConditionType) cephv1.ClusterState {
 	}
 }
 
-// Updating the status of Progressing, Updating or Upgrading to False once cluster is Ready
-func checkConditionFalse(context *clusterd.Context, namespace, name string) {
+// MakeConditionFalse Updates the status of Progressing, Updating or Upgrading to False once cluster is Ready
+func MakeConditionFalse(context *clusterd.Context, namespace, name string) {
 	tempConditionList := []cephv1.ConditionType{cephv1.ConditionUpdating, cephv1.ConditionUpgrading, cephv1.ConditionProgressing}
-	var tempCondition cephv1.ConditionType
+	var tempCondition []cephv1.ConditionType
```

Comment:
nit: instead of using `temp` in the name, how about naming the var `conditionsToUpdate`?

Reply:
Done.